src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,126 @@
+import { Person } from '../types';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { SearchLink } from '../components/SearchLink';
+import { PersonLink } from '../PersonLink/PersonLink';
+import classNames from 'classnames';
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+
+  const isSelected = people.find(person => person.slug === slug);
```

Comment:
The `isSelected` variable is used to determine if a person is selected based on the `slug` parameter. However, it might be more efficient to use a boolean value instead of finding the entire person object. Consider using a boolean expression like `const isSelected = slug && people.some(person => person.slug === slug);` for better performance.